TESTING.md

```diff
@@ -0,0 +1,143 @@
+# How to run phpunit tests
+
+For reference: https://propelorm.org/Propel/cookbook/working-with-unit-tests.html
+
+### 1. Use separate containers because this package requires own database to test its functionality.
+
+Minimal docker configuration for tests:
+
+`docker-compose.yml:`
+
+```yml
+services:
+  php:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - ./src:/var/www
+    depends_on:
+      - mysql
+    networks:
+      - app-network
+    command: tail -f /dev/null
+
+  mysql:
+    image: mysql:8.0
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_DATABASE: php7
+      MYSQL_USER: user
+      MYSQL_PASSWORD: password
+    volumes:
+      - mysql-data:/var/lib/mysql
+    networks:
+      - app-network
+
+networks:
+  app-network:
+    driver: bridge
+
+volumes:
+  mysql-data:
+```
+
+`.env:`
+
+```dotenv
+COMPOSE_PROJECT_NAME=propel1-test-php74
+```
+
+`Dockerfile:`
+
+```Dockerfile
+FROM php:7.4-cli
+
+RUN apt-get update && apt-get install -y \
+    git \
+    unzip \
+    zip \
+    && docker-php-ext-install pdo_mysql
+
+COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
+
+ENV PATH="$PATH:/root/.composer/vendor/bin"
+
+WORKDIR /var/www
+
```

Comment:
I know it's not so useful for these changes but these advices could be useful for next projects, so I suggest these changes
- avoiding using image composer:latest saves to download >150MB
- also we save MB using --no-install-recommends and removing apt-get update cache

more information here https://confluence.wiley.com/pages/viewpage.action?pageId=359403300

```Dockerfile
FROM php:7.4-cli

RUN set -x -o errexit ;\
    apt-get update ;\
    apt-get install -y -q --no-install-recommends \
      git \
      unzip \
      zip \
    ;\
    docker-php-ext-install pdo_mysql ;\
    # cleaning
    apt-get autoremove -y ;\
    apt-get -y clean ;\
    rm -rf \
        /var/lib/apt/lists/* \
        /tmp/* \
        /var/tmp/* \
        /usr/share/doc/*

ARG COMPOSER_VERSION=2.7.7
RUN set -x -o errexit ;\
    php -r "copy('https://raw.githubusercontent.com/composer/getcomposer.org/main/web/installer', 'composer-setup.php');" ;\
    php composer-setup.php --version="${COMPOSER_VERSION}" ;\
    rm -f composer-setup.php ;\
    mv composer.phar /usr/bin/composer ;\
    composer --version

ENV PATH="$PATH:/root/.composer/vendor/bin"

WORKDIR /var/www
```

Reply:
nice!
i just took the simplest config, but yeah, this one is much more optimized, i'll use it